docs/source/usage.rst

```diff
@@ -23,6 +30,24 @@ without the need for the temporary clone:
 
    $ tartufo --repo-path /path/to/my/repo
 
+For ``docker``, mount the local clone to the ``/git`` folder in the docker image:
+
+.. code-block:: sh
+
+   $ docker run --rm -v "/path/to/my/repo:/git" godaddy/tartufo 
+
+When scanning private repositories, the ``docker`` image needs to have access to SSH keys for authorization. Mount the SSH key to ``/tmp/id_rsa`` in the docker image: 
```

Comment:
How about forwarding the SSH agent via the volume (and env vars)?  Something like:

```
docker run --rm -v $SSH_AUTH_SOCK:/agent -e SSH_AUTH_SOCK=/agent ...
```

That will remove the requirement from passing in the key password, means you don't need to run the agent in the container, etc

It adds the requirement that local machine/host needs to be running ssh-agent.

Reply:
Thanks for the suggestion. PR is updated to use ssh-agent forwarding instead of passing keys and passphrase